resources/public/api/conf/1.0/examples/amend_foreign_property_request.json

```diff
@@ -0,0 +1,46 @@
+{
+  "foreignFhlEea": {
+    "income": {
+      "rentAmount": 567.83,
+      "taxDeducted": 4321.92
+    },
+    "expenditure": {
+      "premisesRunningCosts": 4567.98,
+      "repairsAndMaintenance": 98765.67,
+      "financialCosts": 4566.95,
+      "professionalFees": 23.65,
+      "costsOfServices": 4567.77,
+      "travelCosts": 456.77,
+      "other": 567.67,
+      "consolidatedExpenses": 456.98
```

Comment:
Due to the `RULE_BOTH_EXPENSES_SUBMITTED` error I don't believe this request is a valid one